services/QuillLMS/app/models/concept_result.rb

```diff
@@ -104,6 +104,18 @@ def self.parse_extra_metadata(metadata)
       .reject { |_, v| v.nil? || (v.is_a?(Enumerable) && v.empty?) }
   end
 
+  def question_uid = question_uid_from_extra_metadata || question_uid_from_activity
+  def question_uid_from_extra_metadata = extra_metadata&.fetch('question_uid', nil)
+  def question = @question ||= Question.find_by(uid: question_uid)
+
+  # Apparently some ConceptResult records don't have question_uid in their extra_metadata, so we may need to do this complicated lookup
```

Comment:
This is true, but I don't think it should be true for any of the questions for diagnostics completed in the last year and a half (because we started tracking question_uid in [summer 2023](https://github.com/empirical-org/Empirical-Core/pull/10755)).